src/components/ScrobbleItem.tsx

```diff
@@ -166,8 +167,12 @@ export default function ScrobbleItem({
 
   if (scrobble.timestamp) {
     if (!isToday(new Date(scrobble.timestamp))) {
-      timestampFormat = `${settings?.use12Hours ? 'M/d' : 'd/MM'} `;
+      timestampFormat = settings?.use12Hours ? 'M/d' : 'd/MM';
     }
+    if (getYear(new Date(scrobble.timestamp)) < getYear(new Date())) {
+      timestampFormat += '/yyyy';
+    }
+    timestampFormat += ' ';
```

Comment:
These lines should be nested inside the previous `if` (because in that case the year is already irrelevant, and we'd be adding a leading space to the time).